packages/modelviewer.dev/examples/staging-and-camera-control.html

```diff
@@ -302,7 +307,66 @@ <h4>Event listeners can cooperate with camera-controls</h4>
       </div>
     </div>
   </div>
+  <div class="sample">
+    <div id="demo-container-8" class="demo"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="heading">
+          <h2 class="demo-title">Add other interactions like rotating environment map</h2>
+          <h4>Use parameters with rest turntable to rotate only the environment light</h4>
+          <p>By using shift key plus left click mouse or touch pad drag, one can rotate
+			  the environment map to see how the environment background would cast light
+			  onto the object from different angles when it is rotated. Updating orbit to 
+			  new theta value and then reseting the theta value to stop turntable from 
+		  rotating. </p>
+        </div>
+        <example-snippet stamp-to="demo-container-8" highlight-as="html">
+          <template>
+<model-viewer id="envlight-demo" camera-controls src="../shared-assets/models/glTF-Sample-Models/2.0/DamagedHelmet/glTF/DamagedHelmet.gltf"  environment-image="../assets/whipple_creek_regional_park_04_1k.hdr" alt="A 3D model of an astronaut"></model-viewer>
+<script type="module">
+  const modelViewer = document.querySelector("#envlight-demo");
+  
+  // Add the event listeners for mousedown, mousemove, and mouseup
+  modelViewer.addEventListener('mousedown', (event) => {
 
+  	console.log("mouse down");
+  }, true);
+  
+  modelViewer.addEventListener('mousemove', (event) => {
+   
+  // Speed and direction on X axis   
+  var speedX = event.movementX;
+	  
+  if (event.shiftKey && event.which == 1) {
+			
+    const updateOrbit = () => {
+    const orbit = modelViewer.getCameraOrbit(); 
+    console.log(speedX);
+    modelViewer.cameraOrbit = `${orbit.theta + 0.001*(-speedX)}rad ${orbit.phi}rad`;
```

Comment:
you should probably add `auto` to the end of this string for the radius, to make it more clear.

Reply:
Ok will get it resolved in next commit.